### PR TITLE
Run publish operations in parallel

### PR DESCRIPTION
**Pull request by Schwusch**
I got tired of waiting when publishing, and this made publishing 12 times faster on my machine, which is very appreciated :)

### CODE REVIEW

File: generator/lib/package_command.dart [hunk 1]
```diff
@@ -238,52 +240,56 @@ class PublishCommand extends Command {
     final packages = await _scanPackages(config);
 
     final client = Client();
+    final pool = Pool(Platform.numberOfProcessors);
```
Comment (isoos) [inline generator/lib/package_command.dart:243]: Let's use a reasonable max (e.g. 4 or 8).

File: generator/lib/package_command.dart [hunk 2]
```diff
@@ -238,52 +240,56 @@ class PublishCommand extends Command {
...
+              .join('\n'));
+          if (pr.exitCode != 0) {
+            throw Exception('Unable to publish.');
+          }
         }
-      }
+      }));
     }
 
+    await pool.close();
```
Comment (isoos) [inline generator/lib/package_command.dart:292]: Oh, I didn't know that you don't need to wait for the futures to complete.

**Review by isoos — APPROVED** [review]

File: generator/lib/package_command.dart (hunk 2 above)
Comment (Schwusch) [reply generator/lib/package_command.dart:292]: I guess it awaits everything in the queue to be processed before closing?
I just copied what was done in the generate command.

File: generator/lib/package_command.dart (hunk 1 above)
Comment (Schwusch) [reply generator/lib/package_command.dart:243]: How come 4 or 8? And what makes it reasonable?
This is taken from generate command, it seems to work? 乁( •_• )ㄏ

**Pull request merged**